wire-ios-request-strategy/Sources/E2EIdentity/EnrollE2EICertificateUseCase.swift

```diff
@@ -138,16 +138,14 @@ public final class EnrollE2EICertificateUseCase: EnrollE2EICertificateUseCasePro
             clientId: selfClientId ?? "",
             dpopToken: dpopToken)
 
-        let refreshTokenFromCC = try? await enrollment.getOAuthRefreshToken()
-
         let dpopChallengeResponse = try await enrollment.validateDPoPChallenge(
             accessToken: wireAccessToken.token,
             prevNonce: authorizations.nonce,
             acmeChallenge: dPopAuthorization.challenge)
 
         let oidcChallengeResponse = try await enrollment.validateOIDCChallenge(
             idToken: oAuthResponse.idToken,
-            refreshToken: refreshTokenFromCC ?? oAuthResponse.refreshToken,
+            refreshToken: oAuthResponse.refreshToken ?? " ",
```

Comment:
is there a reason to provide a space (`" "`) instead of an empty string (`""`) ?

Reply:
Yes, should be `" "`